implementations/e/applications/ockam_vault_software/lib/vault_software.ex

```diff
@@ -1,48 +1,14 @@
 defmodule Ockam.Vault.Software do
-  @moduledoc """
-  Ockam.Vault.Software
-  """
-
-  use Application
-
   @on_load :load_natively_implemented_functions
 
-  require Logger
-
-  # Called when the Ockam application is started.
-  #
-  # This function is called when an application is started using
-  # `Application.start/2`, `Application.ensure_started/2` etc.
-  #
-  @doc false
-  def start(_type, _args) do
-    Logger.info("Starting #{__MODULE__}")
-
-    # Specifications of child processes that will be started and supervised.
-    #
-    # See the "Child specification" section in the `Supervisor` module for more
-    # detailed information.
-    children = []
-
-    # Start a supervisor with the given children. The supervisor will inturn
-    # start the given children.
-    #
-    # The :one_for_one supervision strategy is used, if a child process
-    # terminates, only that process is restarted.
-    #
-    # See the "Strategies" section in the `Supervisor` module for more
-    # detailed information.
-    Supervisor.start_link(children, strategy: :one_for_one, name: __MODULE__)
-  end
-
   def load_natively_implemented_functions do
-    [:code.priv_dir(:ockam_vault_software), "native", "libockam_elixir_vault_software"]
+    [:code.priv_dir(:ockam), "native", "libockam_elixir_vault_software"]
     |> Path.join()
     |> to_charlist()
     |> :erlang.load_nif(0)
   end
 
-  def default_init do
```

Comment:
the highlighted code should stay the same as it was before this PR and only the new functions added.

Reply:
Fixed